ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/rally/DecryptRallyPayloads.java

```diff
@@ -185,7 +185,7 @@ private void decryptAndReplace(PrivateKey key, JsonNode json, List<JweMapping> m
       }
 
       final PipelineMetadata meta = pipelineMetadataStore.getSchema(attributes);
-      if (meta.jwe_mappings().isEmpty()) {
+      if (meta.jwe_mappings() == null || meta.jwe_mappings().isEmpty()) {
```

Comment:
It looks like Apache Commons has a `MapUtils.isEmpty(meta.jwe_mappings())` that handles the null case in a single call, but this is also fine.

Reply:
I'm going to stick with this, it's easy enough to reason about without pulling in a new import.